styles.css

```diff
@@ -0,0 +1,772 @@
+  left: 13px;
+  z-index: 4;
+  cursor: pointer;
+  width: 50px;
+  height: 50px;
+  opacity: 0;
+}
+
+.menu-wrap .hamburger {
+  position: absolute;
+  top: 5px;
+  left: 13px;
+  z-index: 3;
+  width: 70px;
+  height: 70px;
+  padding: 1rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.menu-wrap .menu {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  visibility: hidden;
+  overflow: hidden;
+  display: flex;
+  z-index: 2;
+}
+
+.display-flex {
+  display: flex;
+  justify-content: space-between;
+}
+
+.concert-wrapper1 > div {
+  background-color: rgba(234, 103, 32, 0.7);
+  bottom: 0;
+  z-index: 4;
+  width: 100%;
+  height: 100%;
+  color: #fffeff;
+}
+
+.concert-wrapper2 > div {
+  background-color: rgba(234, 103, 32, 0.7);
+  bottom: 0;
+  z-index: 4;
+  width: 100%;
+  height: 100%;
+  color: #fffeff;
+}
+
+.menu-wrap .menu > div {
+  background: var(--overlay-color);
+  width: 200vw;
+  height: 400vw;
+  display: flex;
+  align-items: flex-start;
+  justify-content: flex-start;
+  transform: scale(0);
+  padding-top: 106px;
+  transition: margin-right 0.4s ease;
+}
+
+/* Hamburger Line */
+.menu-wrap .hamburger > div {
+  position: relative;
+  flex: none;
+  width: 100%;
+  height: 6px;
+  background: var(--overlay-color);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  transition: all 0.4s ease;
+}
+
+.menu-wrap .menu > div > div {
+  max-width: 90vw;
+  max-height: 100vh;
+  opacity: 0;
+  transition: opacity 0.4s ease;
+}
+
+/* Hamburger Lines - Top & Bottom */
+.menu-wrap .hamburger > div::before,
+.menu-wrap .hamburger > div::after {
+  content: '';
+  position: absolute;
+  z-index: 1;
+  top: -10px;
+  width: 100%;
+  height: 6px;
+  background: inherit;
+}
+
+/* Moves Line Down */
+.menu-wrap .hamburger > div::after {
+  top: 10px;
+}
+
+/* Turns Lines Into X */
+.menu-wrap .toggler:checked + .hamburger > div::before,
+.menu-wrap .toggler:checked + .hamburger > div::after {
+  top: 0;
+  transform: rotate(90deg);
+  background: #fff;
+}
+
+/* Toggler Animation */
+.menu-wrap .toggler:checked + .hamburger > div {
+  transform: rotate(135deg);
+  background: #fff;
+}
+
+.menu-wrap .toggler:checked ~ .menu > div {
+  transform: scale(1);
+  transition-duration: var(--menu-speed);
+}
+
+.menu-wrap .toggler:checked ~ .menu > div > div {
+  opacity: 1;
+  transition: opacity 0.4s ease 0.4s;
+}
+
+/* Rotate On Hover When Checked */
+.menu-wrap .toggler:checked:hover + .hamburger > div {
+  transform: rotate(225deg);
+}
+
+/* Show Menu */
+.menu-wrap .toggler:checked ~ .menu {
+  visibility: visible;
+  opacity: 0.85;
+}
+
+.nav-wrap li {
+  list-style-type: none;
+}
+
+.topbar a {
+  color: #f4f4f4;
+  font-size: 10px;
+}
+
+.nav-wrap li a {
+  text-decoration: none;
+  color: white;
+  font-size: 18px;
+}
+
+#introText {
+  font-size: 20px;
+  font-weight: 400;
+  color: #f46d35;
+  margin-bottom: 0;
+}
+
+.heading {
+  font-size: 50px;
```

Comment:
- [ ] Please decrease the `font-size` of the `h1` element because it isn't matching the design:

    - Your design
    
![hfhf](https://user-images.githubusercontent.com/95412943/219482498-251ad632-7fc4-489e-8f40-c1232b03da4c.PNG)

- requirements design:
![fff](https://user-images.githubusercontent.com/95412943/219482569-20020db9-9c62-411e-8320-37563d19a76b.PNG)